source/array_list.c

```diff
@@ -106,10 +115,12 @@ void aws_array_list_pop_front_n(struct aws_array_list *list, size_t n) {
         size_t popping_bytes = list->item_size * n;
         size_t remaining_items = list->length - n;
         size_t remaining_bytes = remaining_items * list->item_size;
-        memmove(list->data, (uint8_t *)list->data + popping_bytes, remaining_bytes);
+        memmove(
+            list->data, (uint8_t *)list->data + popping_bytes, remaining_bytes);
```

Comment:
This seems weird. Either:
- we should do a line-break for each argument, if we do a line break for any arguments
- or we should try and fit as many args into 80 characters as possible